.github/recipe/recipe.yaml

```diff
@@ -51,15 +47,13 @@ requirements:
     - r-qgg
     - r-quadprog
     - r-quantreg
-    - r-r.utils
```

Comment:
Im not sure where this is used but this sounds like a very basic package?

Reply:
It has no imports any more, so whatever we did use it for is now gone.